src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/routing-1.0.xsd

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<xsd:schema xmlns="http://symfony.com/schema/routing"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="http://symfony.com/schema/routing"
+            elementFormDefault="qualified">
+
+  <xsd:include schemaLocation="http://symfony.com/schema/routing/routing-1.0.xsd" />
```

Comment:
where is the new extended routing scheme going to be hosted? a new location means all the documentation would need updates.

Reply:
Split in #35653.